controls/radrichtextbox/features/merge-documents/merge.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Insert Document at Caret Position
+page_title: Insert Document at Caret Position
+description: Insert Document at Caret Position
+slug: radrichtextbox-features-merge-insert-document
+tags: raddocumentmerger, merge, insert, documents
+published: True
+---
+
+# Insert Document at Caret Position
+
+The [RadDocumentMerger]({%slug radrichtextbox-features-merge-raddocumentmerger%}) class exposes the **InsertDocument() method** that enables you to merge two [RadDocument]({%slug radrichtextbox-features-document-elements-raddocument%}) instances at the caret position of the targeted one. 
```

Comment:
Current Version: ....that enables you to merge two [RadDocument]({%slug radrichtextbox-features-document-elements-raddocument%}) instances at the caret position of the targeted one. 

Suggestion: ...that enables you to merge two [RadDocument]({%slug radrichtextbox-features-document-elements-raddocument%}) instances. The documents will be merged at the caret position of the targeted one.